src/components/PenPreview.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cssmodules from 'react-css-modules';
+import styles from './penpreview.cssmodule.styl';
+import config from 'config';
+
+class PenPreview extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isShowing: false };
+    this.timeoutId = null;
+  }
+  componentWillUpdate(nextProps) {
+    if (nextProps.lineWidth !== this.props.lineWidth &&
+        !this.state.isShowing) {
+      this.show();
+    }
+  }
+  computedSize() {
+    return this.props.lineWidth;
+  }
+  computedStyleForSvg() {
+    return {
+      width: `${this.computedSize()}px`,
+      height: `${this.computedSize()}px`
+    };
+  }
+  computedStyleForCircle() {
+    return {
+      fill: this.props.color
+    };
+  }
+  computedStyleName() {
+    return `svg${(this.state.isShowing ? ' visible' : '')}`;
```

Comment:
@dadaa Yes, but I don't know how to give the effect.

Would you tell me the way to give the effect?